src/containers/Passphrase.jsx

```diff
@@ -43,23 +43,24 @@ const showSuccessThenReload = (t, location) => {
   const ALERT_DURATION = 4000
   Alerter.info(t(translatationKey), { duration: ALERT_DURATION })
 
-  setTimeout(() => {
+  /* setTimeout(() => {
     if (successRedirectUrl) {
       window.location = successRedirectUrl
     } else {
       // the token changes after a password change, so we need to reload
       // the page to get the new one
       window.location.reload()
     }
-  }, ALERT_DURATION)
+  }, ALERT_DURATION) */
```

Comment:
Why did you commit that stuff? I think I know why, but it should not be in this commit

Reply:
It was a test residue 🙁 I remove it from the commit